docs/application/native/guides/connectivity/connectivity-cover-n.md

```diff
@@ -22,6 +22,9 @@ You can connect to a Wireless Local Area Network (WLAN) and transfer data over t
 - [Wi-Fi Direct&reg;](wifi-direct-n.md) **in mobile applications only**
 You can find nearby Wi-Fi Direct devices and discover their details. You can also form a Wi-Fi Direct group to communicate over a peer-to-peer link without wireless access points (base stations) in the infrastructure mode.
```

Comment:
Find a Wi-Fi Direct devices, discover their details and form Wi-Fi Direct group to communicate over a peer-to-peer link without wireless access points (base stations) in the infrastructure mode.